examples/bert.ipynb

```diff
@@ -0,0 +1,274 @@
+        "id": "iaTqNHjpoyWl"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "a9i_RnThVpxU"
+      },
+      "outputs": [],
+      "source": [
+        "import torch\n",
+        "import torch_mlir\n",
+        "import iree_torch\n",
+        "\n",
+        "from transformers import AutoTokenizer, AutoModelForSequenceClassification\n",
+        "\n",
+        "\n",
+        "def prepare_sentence_tokens(hf_model: str, sentence: str):\n",
+        "    tokenizer = AutoTokenizer.from_pretrained(hf_model)\n",
+        "    return torch.tensor([tokenizer.encode(sentence)])\n",
+        "\n",
+        "\n",
+        "class OnlyLogitsHuggingFaceModel(torch.nn.Module):\n",
+        "    \"\"\"Wrapper that returns only the logits from a HuggingFace model.\"\"\"\n",
+        "\n",
+        "    def __init__(self, model_name: str):\n",
+        "        super().__init__()\n",
+        "        self.model = AutoModelForSequenceClassification.from_pretrained(\n",
+        "            model_name,  # The pretrained model name.\n",
+        "            # The number of output labels--2 for binary classification.\n",
+        "            num_labels=2,\n",
+        "            # Whether the model returns attentions weights.\n",
+        "            output_attentions=False,\n",
+        "            # Whether the model returns all hidden-states.\n",
+        "            output_hidden_states=False,\n",
+        "            torchscript=True,\n",
+        "        )\n",
+        "        self.model.eval()\n",
+        "\n",
+        "    def forward(self, input):\n",
+        "        # Return only the logits.\n",
+        "        return self.model(input)[0]\n",
+        "\n",
+        "\n",
+        "# Suppress warnings\n",
+        "import warnings\n",
+        "warnings.simplefilter(\"ignore\")\n",
+        "import os\n",
+        "os.environ[\"TOKENIZERS_PARALLELISM\"] = \"true\""
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# IREE Compilation\n",
+        "\n",
+        "Now, our PyTorch model can be compiled to MLIR and then to a format IREE is able to load and execute."
+      ],
+      "metadata": {
+        "id": "ENzdaB8zogBj"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# The HuggingFace model name to use\n",
+        "model_name = \"philschmid/MiniLM-L6-H384-uncased-sst2\"\n",
+        "\n",
+        "# The sentence to run the model on\n",
+        "sentence = \"The quick brown fox jumps over the lazy dog.\"\n",
+        "\n",
+        "print(\"Parsing sentence tokens.\")\n",
+        "example_input = prepare_sentence_tokens(model_name, sentence)\n",
+        "\n",
+        "print(\"Instantiating model.\")\n",
+        "model = OnlyLogitsHuggingFaceModel(model_name)\n",
+        "\n",
+        "print(\"Tracing model.\")\n",
+        "traced = torch.jit.trace(model, example_input)\n",
```

Comment:
you should be able to omit this and use `use_tracing=True` on torch_mlir.compile

Reply:
Done.